src/Transit/Internal/Peer.hs

```diff
@@ -218,34 +220,34 @@ sendRecord ep record = do
   -- format sz as a fixed 4 byte bytestring
   let payloadSize = toLazyByteString (word32BE (fromIntegral (BS.length record)))
   _ <- sendBuffer ep (toS payloadSize)
-  _ <- sendBuffer ep record
+  _ <- sendBuffer ep (toS record)
```

Comment:
Neither `sendBuffer` not `record` appears to have changed in this branch.  What's the motivation for this `toS`?

Reply:
hmm.. Let me check. I was doing some other changes (making certain bytestrings more lazy instead of strict) and perhaps this got left over. Thanks.